### ISSUE TITLE
Criar estrutura inicial da camada de apresentação (template) no projeto base.

### ISSUE THREAD

**Issue by LucasKauer** [LucasKauer/projeto-base_java#4]

**Comment by LucasKauer** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Criar estrutura inicial da camada de apresentação (template) no projeto base.](https://trello.com/c/hQxfg1wf/11-criar-estrutura-inicial-da-camada-de-apresentacao-template-no-projeto-base)

**Comment by marcelohenrique180** [comment]
@LucasKauer Planejamos usar alguma framework pro front-end (bootstrap, foundation)?

**Comment by LucasKauer** [comment]
Sim! Quem pegar essa issue pode verificar e escolher o meu melhor para o nosso contexto.
